cmd/osquery-perf/agent.go

```diff
@@ -722,11 +722,16 @@ func loadSoftware(platform string, ver string) []map[string]string {
 	}
 
 	var r []map[string]string
-	for _, fi := range software {
+	for i, fi := range software {
+		installedPath := ""
+		if i%2 == 0 {
```

Comment:
Hey @juan-fdz-hawa 
I will put questions in addition to comments so we can later go through them.


Question:
What is this executable for and why are you adding the path every other iteration?

Reply:
It is an 'host' simulator, it allows us to simulate the traffic that a host would normally generate (this is the 'tool' that we use when running load tests). It is added every other iteration because in the real world not all software will return an installed path.

You can read more about it here: https://github.com/fleetdm/osquery-perf